lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -306,6 +306,13 @@ findDatabases tr dir = do
   where
     expectedPrefix = T.pack $ keyTypeDescriptor $ Proxy @k
 
+-- | A data-type for capturing column status. Used to be represented as a
+-- 'Maybe Bool' which is somewhat confusing to interpret.
```

Comment:
Suggestion: remove the line starting with "Used to be represented...".

Justification: the new code is already very clear. :+1: 

(We don't need to remind the reader how confusing it used to be :smile_cat:)

```suggestion
-- | Indicates whether or not a column and its parent table are present
--   in the database.
```